internal/services/containers/probe.go

```diff
@@ -100,4 +100,13 @@ func SchemaContainerGroupProbe() *pluginsdk.Schema {
 			},
 		},
 	}
+	if features.FourPointOh() {
+		httpGetSchema := schema.Elem.(*pluginsdk.Resource).Schema["http_get"]
+		httpGetSchema.Elem.(*pluginsdk.Resource).Schema["scheme"].ValidateFunc = validation.StringInSlice([]string{
+			"http",
+			"https",
+		}, false)
+		schema.Elem.(*pluginsdk.Resource).Schema["http_get"] = httpGetSchema
+	}
+	return schema
```

Comment:
Rather than doing this, can we update the Validate func to use the Constant values - and then use ignore the casing on those until 4.0?